From db02ae92dcacab17fb2e40b36707aa4c1ee88a31 Mon Sep 17 00:00:00 2001
From: Paul Pettit <p.pettit@keele.ac.uk>
Date: Tue, 2 Aug 2016 10:52:03 +0100
Subject: [PATCH] Fix test for blank lis_person_sourcedid

---
 .../opencastproject/kernel/security/LtiLaunchAuthenticationHandler.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/matterhorn-kernel/src/main/java/org/opencastproject/kernel/security/LtiLaunchAuthenticationHandler.java b/modules/matterhorn-kernel/src/main/java/org/opencastproject/kernel/security/LtiLaunchAuthenticationHandler.java
index 0df5b64..0b91fb4 100644
--- a/modules/matterhorn-kernel/src/main/java/org/opencastproject/kernel/security/LtiLaunchAuthenticationHandler.java
+++ b/modules/matterhorn-kernel/src/main/java/org/opencastproject/kernel/security/LtiLaunchAuthenticationHandler.java
@@ -153,7 +153,7 @@ public class LtiLaunchAuthenticationHandler implements
       //If supplied we use the human readable name
       String suppliedEid = request.getParameter("lis_person_sourcedid");
       //This is an optional field it could be null
-      if (suppliedEid != null) {
+      if (StringUtils.isNotBlank(suppliedEid)) {
         userIdFromConsumer = suppliedEid;
       } else {
         //if no eid is set we use the supplied ID
--
2.9.2
